tests/pipelines/test_pipelines_common.py

```diff
@@ -34,23 +40,33 @@
     IBertConfig,
     RobertaConfig,
     TextClassificationPipeline,
+    TFAutoModelForSequenceClassification,
     pipeline,
 )
 from transformers.pipelines import PIPELINE_REGISTRY, get_task
 from transformers.pipelines.base import Pipeline, _pad
 from transformers.testing_utils import (
+    TOKEN,
+    USER,
     CaptureLogger,
     is_pipeline_test,
+    is_staging_test,
     nested_simplify,
     require_scatter,
     require_tensorflow_probability,
     require_tf,
     require_torch,
     slow,
 )
+from transformers.utils import is_tf_available, is_torch_available
 from transformers.utils import logging as transformers_logging
 
 
+sys.path.append(str(Path(__file__).parent.parent.parent / "utils"))
```

Comment:
I am always super uneasy about adding this stuff.

Why can't we have the `PairClassificationPipeline` defined directly in the tests ?

Reply:
It needs to be in a separate module for the feature to work.